applications/kommander-flux/2.6.2/helmrelease/helmrelease.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: flux2
+  namespace: ${releaseNamespace}
+spec:
+  interval: 1m
+  url: "${ociRegistryURL:=oci://ghcr.io}/fluxcd-community/charts/flux2"
+  ref:
+    tag: 2.16.1
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: kommander-flux
+  namespace: ${releaseNamespace}
+spec:
+  chartRef:
+    kind: OCIRepository
+    name: flux2
+    namespace: ${releaseNamespace}
+  interval: 15s
+  install:
+    crds: CreateReplace
+    remediation:
+      retries: 30
+    createNamespace: true
+  upgrade:
```

Comment:
lets use 
```
uninstall
  deletionPropagation: orphan
```
This is just being defensive as it should be unused if remediation doesn't kick in.